src/test/java/com/cloudant/tests/util/Utils.java

```diff
@@ -33,4 +44,58 @@ public static String getHostName(String account) {
         }
 
     }
+
+    public static void removeReplicatorTestDoc(CloudantClient account, Response replicatorResponse)
+            throws Exception {
+
+        //Find replicator document for removal
+        ReplicatorDocument replicatorDoc = waitForReplicatorToComplete(account, replicatorResponse);
+
+        Response removeResponse = account.replicator()
+                .replicatorDocId(replicatorDoc.getId())
+                .replicatorDocRev(replicatorDoc.getRevision())
+                .remove();
+
+        assertThat(removeResponse.getError(), is(nullValue()));
+    }
+
+    public static ReplicatorDocument waitForReplicatorToComplete(CloudantClient account, Response
```

Comment:
`removeReplicatorTestDoc` and `waitForReplicatorToComplete` should just take the document ID I think, as that's the only attribute that's used -- prefer passing the smallest possible amount of information to a method so it's clear what its dependencies are.